src/storage/mod.rs

```diff
@@ -176,14 +175,12 @@ impl Display for Command {
                 ref ctx,
                 ref keys,
                 start_ts,
-                for_update_ts,
                 ..
             } => write!(
                 f,
-                "kv::command::acquirepessimisticlock keys({}) @ {},{} | {:?}",
+                "kv::command::acquirepessimisticlock keys({}) @ {} | {:?}",
```

Comment:
Better to display for_update_ts, it is useful for debugging.

Reply:
Done